x-pack/test_serverless/functional/test_suites/search/playground_overview.ts

```diff
@@ -5,23 +5,167 @@
+  let roleAuthc: RoleCredentials;
+
+  describe('Serverless Playground Overview', () => {
+    let closeOpenAIConnector: () => Promise<void> | undefined;
 
-export default function ({ getPageObjects }: FtrProviderContext) {
-  const pageObjects = getPageObjects(['svlCommonPage', 'svlCommonNavigation', 'svlPlaygroundUI']);
-  describe('Playground', function () {
     before(async () => {
       await pageObjects.svlCommonPage.login();
-      await pageObjects.svlCommonNavigation.sidenav.clickLink({ deepLinkId: 'searchPlayground' });
+      await pageObjects.svlCommonNavigation.sidenav.clickLink({
+        deepLinkId: 'searchPlayground',
+      });
+
+      const requestHeader = svlCommonApi.getInternalRequestHeader();
+      roleAuthc = await svlUserManager.createApiKeyForRole('admin');
+      openAIConnectorGen = createOpenAIConnector({
+        configService,
+        supertest: supertestWithoutAuth,
+        requestHeader,
+        apiKeyHeader: roleAuthc.apiKeyHeader,
+      });
+
+      closeOpenAIConnector = (await openAIConnectorGen.next()).value;
     });
 
     after(async () => {
+      await closeOpenAIConnector?.();
+      await svlUserManager.invalidateApiKeyForRole(roleAuthc);
       await pageObjects.svlCommonPage.forceLogout();
     });
 
-    it('playground app is loaded', async () => {
-      await pageObjects.svlPlaygroundUI.PlaygrounStartChatPage.expectPlaygroundStartChatPageComponentsToExist();
-      await pageObjects.svlPlaygroundUI.PlaygrounStartChatPage.expectPlaygroundHeaderComponentsToExist();
+    after(async () => {
+      await esArchiver.unload(esArchiveIndex);
+    });
+
+    describe('start chat page', () => {
+      it('playground app is loaded', async () => {
+        await pageObjects.searchPlayground.PlaygroundStartChatPage.expectPlaygroundStartChatPageComponentsToExist();
+        await pageObjects.searchPlayground.PlaygroundStartChatPage.expectPlaygroundHeaderComponentsToExist();
+      });
+
+      it('show no index callout', async () => {
+        await pageObjects.searchPlayground.PlaygroundStartChatPage.expectNoIndexCalloutExists();
+        await pageObjects.searchPlayground.PlaygroundStartChatPage.expectCreateIndexButtonToMissed();
+      });
+
+      it('hide no index callout when index added', async () => {
+        await createIndex();
+        await pageObjects.searchPlayground.PlaygroundStartChatPage.expectSelectIndex(indexName);
+      });
+
+      it('show add connector button', async () => {
+        await pageObjects.searchPlayground.PlaygroundStartChatPage.expectAddConnectorButtonExists();
+      });
+
+      it('click add connector button opens connector flyout', async () => {
+        await pageObjects.searchPlayground.PlaygroundStartChatPage.expectOpenConnectorPagePlayground();
+      });
+
+      it('hide gen ai panel when connector exists', async () => {
+        await pageObjects.searchPlayground.PlaygroundStartChatPage.expectHideGenAIPanelConnector(
+          createConnector
```

Comment:
The first call is creating simulator that use proxy port. And then on the second call we call API for creating connector. But yeah now I see it might be confusing. 
As we don't mock openAI reply because of ActionsClientIssue we might also avoid using simulator and create connector with any URL

Reply:
Updated creating connector function